### PR TITLE
Add logrocket

### PR DESCRIPTION
**Pull request by solovev**

### CODE REVIEW

File: packages/web/.env [hunk 1]
```diff
@@ -12,3 +12,5 @@ NEXT_PUBLIC_FEATURE_PP_PAGE=true
 # API
 NEXT_PUBLIC_GRAPH_URL=http://localhost:4000
 
+# Monitoring
+NEXT_PUBLIC_LOGROCKET_APP_ID=
```
Comment (LeoVS09) [inline packages/web/.env:16]: Maybe better move to `.env.example`?

File: packages/web/app/providers/wallet/wallet-wrapper-provider.tsx [hunk 2]
```diff
@@ -56,33 +59,45 @@ const WalletWrapperImplementationProvider: React.FC<Props> = ({ children }) => {
...
+    if (success)
       await W3O.autoSelectProperChain(chain, chains, setOnboardChain);
-    
   }, [chain, chains, onboardConnect, setOnboardChain]);
 
   const disconnect = React.useCallback(async () => {
-    if (wallet) 
-      await W3O.disconnect(wallet.label, onboardDisconnect);
-    
+    if (wallet) await W3O.disconnect(wallet.label, onboardDisconnect);
```
Comment (LeoVS09) [inline packages/web/app/providers/wallet/wallet-wrapper-provider.tsx:100]: Eh, the linter must not allow such code.

File: packages/web/release-logrocket.cjs [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+     * Remove source map (*.map files) from the build.
+     */
+    await asyncExec(`find ./${BUILD_PATH} -name '*.map' -delete`);
+
+    /**
+     * Clear links to the source maps from the source code.
+     */
+    await asyncExec(
+      `find ./${BUILD_PATH} -regex '.*.\(js\|css\)' -exec sed -i -E '/^\/\/# sourceMappingURL.*/d' {} +`
+    );
```
Comment (LeoVS09) [inline packages/web/release-logrocket.cjs:60]: Why do we need to remove source maps?

File: packages/web/release-logrocket.cjs [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+}
+
+/**
+ * Make LogRocket release only on production/staging/development builds (or if the commit message has "logrocket" substring).
+ */
+function isReleaseRequired() {
+  return (
+    VARS.ENV_NAME !== "preview" ||
+    VARS.BRANCH_NAME.toLowerCase() === "staging" ||
+    VARS.COMMIT_MESSAGE.toLowerCase().includes("logrocket")
```
Comment (LeoVS09) [inline packages/web/release-logrocket.cjs:73]: Are `development` not included in this?

**Review by LeoVS09 — APPROVED** [review]

File: packages/web/release-logrocket.cjs (hunk 3 above)
Comment (solovev) [reply packages/web/release-logrocket.cjs:60]: nextjs will serve it otherwise

File: packages/web/release-logrocket.cjs (hunk 4 above)
Comment (solovev) [reply packages/web/release-logrocket.cjs:73]: it should be included, `VARS.ENV_NAME !== "preview"`

File: packages/web/.env (hunk 1 above)
Comment (solovev) [reply packages/web/.env:16]: hm, tbh, don't see a reason to create a new file just for this

File: packages/web/app/providers/wallet/wallet-wrapper-provider.tsx (hunk 2 above)
Comment (solovev) [reply packages/web/app/providers/wallet/wallet-wrapper-provider.tsx:100]: yeah, it should wrap it by curly braces :))

**Pull request merged**